apps/public-docsite-v9/src/Concepts/Upgrade/FromV8/Components/Textarea.stories.mdx

```diff
@@ -0,0 +1,111 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Concepts/Upgrading/from v0/Components/Textarea Upgrade" />
+
+# Textarea Upgrade
+
+Fluent UI v8 provides the `TextField` control to allow users to enter and edit text. Fluen UI v9 provides a `Textarea` control, but has a different API.
+
+v9 has takes a different approach on the `TextField` component and uses the `Input` and `Textarea` components. `Input` in this case is the general `TextField` component and `Textarea` is a `TextField` with the `multiline` prop set to `true`.
```

Comment:
_nit:_ wording

```suggestion
v9 takes a different approach than v8 with respect to text inputs, and instead of using a single `TextField` component instaed uses separate `Input` and `Textarea` components. v9's `Input` in this case is equivalent to the general v8's `TextField`, while v9's `Textarea` is equivalent to v8's `TextField` with the `multiline` prop set to `true`.
```